cheatsheets/Server_Side_Request_Forgery_Prevention_Cheat_Sheet.md

```diff
@@ -360,19 +385,27 @@ def is_private_ip(ip_address):
     return is_private   
 ```
 
-#### Network layer
+##### Network layer
 
 Similar to the following [section](Server_Side_Request_Forgery_Prevention_Cheat_Sheet.md#network-layer).
 
-# References
+## IMDSv2 in AWS
+
+In cloud enviroments SSRF is often used to access and steal credentaials and access tokens from metadata services (e.g. AWS Instance Metadata Service, Azure Instance Metadata Service, GCP metadata server).
+
+[IMDSv2](https://aws.amazon.com/blogs/security/defense-in-depth-open-firewalls-reverse-proxies-ssrf-vulnerabilities-ec2-instance-metadata-service/) is an additional defence-in-depth mechanism for AWS that mitigates some of the instances of SSRF. To leverage this protection migrate to IMDSv2 and disable old IMDSv1. 
```

Comment:
I'd recommend to set the `To leverage ...` on a new line, and link IMDSv2 to this link: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instancedata-data-retrieval.html
What do you think?

Reply:
Sure! I will update it :)